CHANGELOG.md

```diff
@@ -7,18 +7,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.5.0] - 2022-07-21
+
 ### Changed
 
 - Update `xtra` to [upstream](https://github.com/Restioson/xtra). This involved re-implementing some of the features
   in [comit-network's fork](https://github.com/comit-network/xtra) internally. Xtra message handler metrics were also
   removed in favour of the new `instrumentation` feature combined with
   [Grafana Tempo's span metrics](https://grafana.com/docs/tempo/latest/server_side_metrics/span_metrics/).
+- Allow rollovers from previous versions for takers to make the rollover protocol more resilient against the maker being ahead and avoid consecutive rollover failure.
+
+### Removed
+
+- The initial tour through ItchySats the underlying library caused issues that could not be overcome.
 
 ### Added
 
 - Add new argument to the maker: `ignore-migration-errors`. If enabled, the maker will start if an error occurred when opening the database, if not, it will fail fast. This can come handy to prevent accidentally creating a new empty database in case database migration was unsuccessful.
 - New metrics: the maker tracks how many offers have been sent (`offer_messages_sent_total`) and the taker tracks how many offers have been received (`offer_messages_received_total`).
 - Allow taker to provide extended private key as argument when starting. This key will be used to derive the internal wallet according to (Bip84)[https://github.com/bitcoin/bips/blob/master/bip-0084.mediawiki].
+- Support for tokio console by for debugging. It can be activated with `--tokio-console`.
+- Show the daemon-version in the info box of the taker-ui.
+
```

Comment:
I think we also added traces

```suggestion
- add support for instrumented traces for debugging. It can be activated with ``--instrumented`. 

```